server/routes/userRequest.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const router = express.Router();
+const UserRequest = require('../../db/UserRequest.js');
+
+//get a single user request
+router.get('/homes/:request', (req, res) => {
+  let data = req.body.data;
+  let phoneNumber = Number(data.phone);
+  UserRequest.findOne({ phone: phoneNumber })
+  .then(data => res.send(data))
+  .catch(error => res.sendStatus(500));
+})
+
+//get all user requests
+router.get('/homes/all', (req, res) => {
```

Comment:
Here as well can be changed
```suggestion
router.get('/homes', (req, res) => {
```